test/e2e/specs/specs-playwright/wp-start__importer.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @group calypso-pr
+ */
+
+import { setupHooks, DataHelper, LoginPage, StartImportFlow } from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( 'Site Import' ), function () {
+	let page: Page;
+	let startImportFlow: StartImportFlow;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+		startImportFlow = new StartImportFlow( page );
+	} );
+
+	it( 'Log in', async function () {
+		const loginPage = new LoginPage( page );
+		await loginPage.login( { account: 'defaultUser' } );
+	} );
+
+	describe( 'Follow the WordPress flow', function () {
+		it( 'Navigate to Capture page', async function () {
+			await page.goto(
+				DataHelper.getCalypsoURL( '/start/importer', { siteSlug: 'e2eflowtesting4.wordpress.com' } )
```

Comment:
I do not see an associated change in `DataHelper` to implement the `siteSlug` param. Is this done elsewhere?

Reply:
I didn't know there needed to be anything added here: the second parameter of `getCalypsoURL()` appears to be an unrestricted blob of query string key/value pairs.